src/plugins/jartic/data_parser.py

```
@@ -0,0 +1,539 @@
+                            content = f.read().decode('utf-8', errors='ignore')
+
+                            if file_name.endswith('.json'):
+                                locations.extend(
+                                    self._parse_json_locations(content, file_name)
+                                )
+                            elif file_name.endswith('.csv'):
+                                locations.extend(
+                                    self._parse_csv_locations(content, file_name)
+                                )
+
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)